docs/diagrams/DoneCryptoSequenceDiagram.puml

```diff
@@ -0,0 +1,116 @@
+@startuml
+'https://plantuml.com/class-diagram
+!include Styles.puml
+
+box console COLOR_BRIGHT_GREEN
+participant ":InputParser" as InputParser COLOR_CONSOLE
+end box
+
+box command COLOR_YELLOW
```

Comment:
For the large boxes can we choose the like similar color to the class they represent. But use a like a very faint color or something. You may need to use a color picker and define a new color. Same for console and model. UI already looks good so can stick with it

If not possible we can just stick with white and KIV